### PR TITLE
cherry-pick linting fixes

### PR DESCRIPTION
**Pull request by paulbalaji**
#### Description

- update broken links
- remove djangogirls link

#### Tests

- copied across from `docs/pipeline` branch

#### Documentation

n/a

#### Primary reviewers
If your change will take a long time to review, you can name at most two primary reviewers who are ultimately responsible for reviewing this request. @ mention them.

### CODE REVIEW

File: docs/projects/blank/overview.md [hunk 1]
```diff
@@ -18,4 +18,4 @@ It has the minimum GDK feature set you need to start developing games for Spatia
 - [Worker build configuration]({{urlRoot}}/projects/myo/build#build-your-workers).
 - A set of Unity Scenes to use in development.
 
-> **Note:** Whichever route you choose, you need basic proficiency in [programming in Unity (Unity documentation)](https://unity3d.com/programming-in-unity), version control systems (such as [Git (github.io)](https://try.github.io/)), and [command line interfaces (djangogirls.org)](https://tutorial.djangogirls.org/en/intro_to_command_line/) to use the SpatialOS GDK for Unity.
+> **Note:** Whichever route you choose, you need basic proficiency in [programming in Unity (Unity documentation)](https://unity3d.com/programming-in-unity), version control systems (such as [Git (github.io)](https://try.github.io/)), and command line interfaces (CLI) to use the SpatialOS GDK for Unity.
```
Comment (jamiebrynes7) [inline docs/projects/blank/overview.md]: While you're here can we get rid of the (domain) notation for external links

File: docs/projects/fps/overview.md [hunk 2]
```diff
@@ -6,8 +6,8 @@
...
 
 To dive right in, follow the [Get started]({{urlRoot}}/projects/fps/get-started/get-started) guide which uses the FPS Starter Project. Once you have completed this, you can follow the _Add your first feature_ guide which is a Health Pick-up tutorial, following on from the _Get started_ guide.
 
-> **Note:** Whichever route you choose, you need basic proficiency in [programming in Unity (Unity documentation)](https://unity3d.com/programming-in-unity), version control systems (such as [Git (github.io)](https://try.github.io/)), and [command line interfaces (djangogirls.org)](https://tutorial.djangogirls.org/en/intro_to_command_line/) to use the SpatialOS GDK for Unity.
+> **Note:** Whichever route you choose, you need basic proficiency in [programming in Unity (Unity documentation)](https://unity3d.com/programming-in-unity), version control systems (such as [Git (github.io)](https://try.github.io/)), and command line interfaces (CLI) to use the SpatialOS GDK for Unity.
```
Comment (paulbalaji) [inline docs/projects/fps/overview.md]:
```suggestion
> **Note:** Whichever route you choose, you need basic proficiency in [programming in Unity](https://unity3d.com/programming-in-unity), version control systems (such as [Git](https://try.github.io/)), and command line interfaces (CLI) to use the SpatialOS GDK for Unity.
```

File: docs/projects/fps/get-started/get-started.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ Once you have completed this introduction, you are then ready to learn how to bu
 We recommend starting with the FPS Starter Project by going to the next step: [Set up]({{urlRoot}}/projects/fps/get-started/set-up.md), but you can get going with the [Blank Starter Project]({{urlRoot}}/projects/blank/overview) if you prefer this route.
 <br/>
 
-> **Note:** Whichever route you choose, you need basic proficiency in [programming in Unity](https://unity3d.com/programming-in-unity), version control systems (such as [Git](https://try.github.io/)), and [command line interfaces](https://tutorial.djangogirls.org/en/intro_to_command_line/) to use the SpatialOS GDK for Unity.
+> **Note:** Whichever route you choose, you need basic proficiency in [programming in Unity (Unity documentation)](https://unity3d.com/programming-in-unity), version control systems (such as [Git (github.io)](https://try.github.io/)), and command line interfaces (CLI) to use the SpatialOS GDK for Unity.
```
Comment (paulbalaji) [inline docs/projects/fps/get-started/get-started.md]:
```suggestion
> **Note:** Whichever route you choose, you need basic proficiency in [programming in Unity](https://unity3d.com/programming-in-unity), version control systems (such as [Git](https://try.github.io/)), and command line interfaces (CLI) to use the SpatialOS GDK for Unity.
```

File: docs/projects/blank/overview.md (hunk 1 above)
Comment (paulbalaji) [inline docs/projects/blank/overview.md]:
```suggestion
> **Note:** Whichever route you choose, you need basic proficiency in [programming in Unity](https://unity3d.com/programming-in-unity), version control systems (such as [Git](https://try.github.io/)), and command line interfaces (CLI) to use the SpatialOS GDK for Unity.
```

**Pull request merged**